adr/2022-08-decouple-from-ietf.md

```diff
@@ -0,0 +1,82 @@
+# Decoupling from IETF
+
+* Status: accepted
+* Deciders: @jdesrosiers @relequestual @awwright
```

Comment:
Has this decision already been made?  I guess I'm a bit confused about the ADR process.  If this is a done deal and all this PR is concerned with is the write-up, then my comments probably don't apply.

Reply:
I wrote this up because @Relequestual asked me to, but I agree that it feels awkward in that I don't feel like there was ever a point where a decision was made.

I'm viewing this PR as a proposed resolution that isn't a "done deal" until it's merged. At this point, I think there's more than enough consensus to write up this PR, but not yet enough to merge it. The PR is a sort of "last call" for people to express support or dissent before a final decision is made.